### PR TITLE
Added classes to support observable:deviceType

### PR DESCRIPTION
**Pull request by b0bkaT**
This Pull Request resolves all requirements of Issue #434 .

# Coordination

- [x] Pull Request is against correct branch
- [x] Pull Request is in, or reverted to, Draft status before Solutions Approval vote has passed.
- [x] CI passes in UCO feature branch
- [x] CI passes in UCO current `unstable` branch ([c5618b4](https://github.com/ucoProject/UCO-Archive/commit/c5618b41ac41063d9f7bdd2047c2b617ab575b59))
- [x] CI passes in CASE current `unstable` branch tracking UCO's `unstable` as submodule ([f3ccdc8](https://github.com/casework/CASE-Archive/commit/f3ccdc8da6de69e5416d77fd872180525af8cca2))
- [x] Impact on SHACL validation [reviewed](https://github.com/casework/CASE-Examples/commit/fbc7b2f882951a349e9b9c89441de9e4cc112cad) for CASE-Examples
- [x] Impact on SHACL validation remediated for CASE-Examples *(N/A)*
- [x] Impact on SHACL validation [reviewed](https://github.com/casework/casework.github.io/commit/2f8e824e13322a10c09c6fc59e268ebb27dd5bd1) for casework.github.io
- [x] Impact on SHACL validation remediated for casework.github.io *(N/A)*
- [x] Milestone linked
- [x] Solutions Approval vote logged on corresponding Issue (once logged, can be taken out of Draft PR status)

### CODE REVIEW

File: tests/examples/message_thread_XFAIL_validation.ttl
Comment (ajnelson-nist) [inline tests/examples/message_thread_XFAIL_validation.ttl:10]: For everyone's awareness, this Git noise appears to be due to a new effect in `pyshacl` after `rdflib`'s 6.2.0 release.  It is a mild nuisance for Git review, but otherwise harmless.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:3697]: This change is out of scope of the proposal, but is sufficiently minor I'm fine with it remaining.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:3701]: Committee decision point - capital "I", or lowercase?

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:6189]: And here is one reason we need to be careful with the name `observable:Computer` - the definition on `Computer` currently aligns that concept with form factors that would be found on or near a desk,.  A tablet is not appropriately a subclass of a class bearing that description, but it is appropriately a subclass of a more general "Computer".  `observable:Computer` should be distinguished from `observable:Device`.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:7042]: Considering CCTV systems, we might want to leave 'audio' out of scope of this concept.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:7053]: Same remark here on 'audio'.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [inline ontology/uco/observable/observable.ttl:9046]: This Facet needs a name correction.

(Sorry if this is a repeat, I remember typing this once already...)

**Review by ajnelson-nist — CHANGES_REQUESTED** [review]
@b0bkaT , please address and close out the conversation threads requesting changes, and leave the more informational ones unresolved so the committee can see them.

File: ontology/uco/observable/observable.ttl
Comment (sbarnum) [reply ontology/uco/observable/observable.ttl:1103]: I would agree that Computer is more than just a desktop computer.
I would suggest changing the defintion and maybe coming up with some subclasses for types of computers one of which could be for "desktop" type thing.
Also, this class likely collides with existing ComputerSpecification class

File: tests/examples/message_thread_XFAIL_validation.ttl
Comment (ajnelson-nist) [reply tests/examples/message_thread_XFAIL_validation.ttl]: This should remain cleaned out, per [PR 456](https://github.com/ucoProject/UCO/pull/456).

**Comment by b0bkaT** [comment]
I've implemented the changes as discussed a few days ago.

File: ontology/uco/observable/observable.ttl
Comment (sbarnum) [reply ontology/uco/observable/observable.ttl:6801]: And a video camera does not necessarily capture anything in digital memory.
For that it seems like there would need to be a DigitalRecordingVideoCamera subclass of a DigitalVideoCamera subclass of VideoCamera.
Some video cameras are analog and some only stream and do not record.

**Comment by ajnelson-nist** [comment]
Converting to Draft until Solutions Approval vote is logged.

File: ontology/uco/observable/observable.ttl
Comment (plbt5) [reply ontology/uco/observable/observable.ttl:3558]: Since it's considered a Class, and an (mnemonic) IRI as opposed to a name, follow the rules for IRI's and Classes, viz using the capital "I". In the comment and label fields, since these are human readable only, apply the usual name of the device, viz with a lowercase "i".

Comment (plbt5) [reply ontology/uco/observable/observable.ttl:3558]: When we have come to an agreement, consolidate this design decision by adding it to the Design Document.

Comment (ajnelson-nist) [reply ontology/uco/observable/observable.ttl:3558]: We'll go with uppercase.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [reply ontology/uco/observable/observable.ttl:5959]: I believe this is resolved.

File: ontology/uco/observable/observable.ttl
Comment (ajnelson-nist) [reply ontology/uco/observable/observable.ttl:6801]: We will wait until after 1.0.0 to integrate this one.  Cutting for now.

Comment (ajnelson-nist) [reply ontology/uco/observable/observable.ttl]: Cut in [1316130](https://github.com/ucoProject/UCO/commit/1316130).

**Review by ajnelson-nist — APPROVED** [review]
Good to merge once CI passes.

**Pull request merged**
